modules/installation-user-infra-exporting-common-variables-arm-templates.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-user-infra.adoc
+
+[id="installation-user-infra-exporting-common-variables-arm-templates_{context}"]
+= Exporting common variables for ARM templates
+
+You must export a common set of variables that are used with the provided Azure
+Resource Manager (ARM) templates used to assist in completing a user-provided
+infrastructure install on Microsoft Azure.
+
+[NOTE]
+====
+Specific ARM templates can also require additional exported variables, which are
+detailed in their related procedures.
+====
+
+.Prerequisites
+
+* Obtain the {product-title} installation program and the pull secret for your cluster.
+* Generate the Ignition config files for your cluster.
+* Install the `yq` package.
+
+.Procedure
+
+* Export the following common variables to be used by the provided ARM
+templates:
++
+----
+$ export KUBECONFIG=<installation_directory>/auth/kubeconfig <1>
+$ export CLUSTER_NAME=`yq -r .metadata.name install-config.yaml`
```

Comment:
We can't export those variables, because the `install-config.yaml` file already has been consumed by the OCP installer when you run the `./openshift-install create manifests`.